.github/workflows/labeler.yml

```
@@ -1,16 +1,14 @@
 name: "Pull Request Labeler"
 on:
-  pull_request:
-    types: [opened, synchronize]
-    paths:
-      - 'content/**'
-    branches:
-      - '**'
+- pull_request_target
```

Amending to @yunkon-kim's [suggestion](https://github.com/cncf/glossary/pull/1286/files#r962308875):

```suggestion
  pull_request_target:
    types: [opened, synchronize]
    paths:
      - 'content/**.md'
    branches:
      - 'dev-**'
      - 'main'
```

If we add 'main' as target branch, then this will automatically label `lang/en` for the PRs that add/modify 'content/en/**.md' and target `main`!

(Ref: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request-1)

---

For information:
The default value of `sync-labels` is `false`. (https://github.com/actions/labeler#inputs)
We might need to add [`action.yml`](https://github.com/actions/labeler/blob/main/action.yml) to this repo and set the value of `sync-labels` to `true`, 
to make `synchronize`-related thing work.

---

@jihoon-seo thanks for your updates!

I agree that `main` is a vital point in this workflow.

For a simple discussion:
I was thinking about the `synchronize` type. However, I couldn't find any cases of reassigning the label when synchronizing.

Also, when we add the `synchronize` type, this workflow will be triggered by every PR synchronization. 

Therefore, it would be nice to add the 'synchronize' as needed. 

If anyone has a case that `synchronize` is necessary, please feel free to share.

---

I removed "synchronize" for this time. Let's see it is necessary after some period. :)

---

@seokho-son 

`main` was not added to the `branches` section. Is it ok?

---

@yunkon-kim Added :)